src/python/grapl_analyzerlib/.ipynb_checkpoints/Untitled-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
+ "cells": [],
+ "metadata": {},
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
we should .gitignore *.ipynb most likely

Reply:
Crap, I thought I hadn't pushed this. Agreed.